modules/apim_analytics_worker_master/manifests/params.pp

```diff
@@ -0,0 +1,95 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2019 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+
+# Class apim_analytics_dashboard::params
```

Comment:
`apim_analytics_dashboard` should be `apim_analytics_worker`

Reply:
Resolved https://github.com/wso2/puppet-apim/pull/88/commits/923e24c8610696246c962d255a32f5decd0017e4